metron-analytics/metron-profiler-client/src/test/resources/core-site.xml

```diff
@@ -0,0 +1,99 @@
+  <configuration>
+    
+    <property>
+      <name>fs.defaultFS</name>
+      <value>hdfs://node1:8020</value>
```

Comment:
I think this is the cause of the unit test failures.  We really shouldn't have a core-site.xml here.  What's caused it to be required?